extension/lib/main.js

```diff
@@ -113,14 +110,65 @@ exports.main = function (options, callbacks) {
         widget.tooltip = config.extension.widget_tooltips[data];
         break;
     }
+  });*/
+
+  var { Frame } = require("sdk/ui/frame");
+  var { ToggleButton } = require('sdk/ui/button/toggle');
+  var { Toolbar } = require("sdk/ui/toolbar");
+
+  var frame = new Frame({
+    url: "./frame/frame.html"
   });
 
+  var menuButton = ToggleButton({
+    id: "menu-button",
+    label: "Menu",
+    icon: {
+      "32" : "./addon32.png",
+      "64" : "./addon64.png"
```

Comment:
Personally I find the sizes of the elements compared to the rest of the browser way too large. @kwierso is all that expected or is there still an ongoing ui refactoring? Right now it's really not that shiny.